intermediate_html_css/intermediate_html_concepts/svgs.md

```diff
@@ -86,22 +94,16 @@ For now, just do whichever works best for your use-case. Linking is generally cl
 
--   [What is the `xmlns` attribute?](#anatomy-of-an-svg)
--   [What are some situations where you _wouldn't_ want to use SVG?](#drawbacks)
--   [What are the benefits of "inlining" your SVGs? What are the drawbacks?](#embedding-svgs)
+- [What is the `xmlns` attribute?](#anatomy-of-an-svg)
+- [What are some situations where you *wouldn't* want to use SVG?](#drawbacks)
+- [What are the benefits of "inlining" your SVGs? What are the drawbacks?](#embedding-svgs)
 
 ### Additional resources
 
-This section contains helpful links to related content. It isn’t required, so consider it supplemental.
-
-1.  There are lots of great free SVG icon libraries. A few worth checking out: [Material icons](https://fonts.google.com/icons), [Feather icons](https://feathericons.com/), [The Noun Project](https://thenounproject.com/term/free/), and [Ionicons](https://ionic.io/ionicons).
-2.  If you want a deep-dive into the details of SVGs and their elements, the [MDN tutorial](https://developer.mozilla.org/en-US/docs/Web/SVG/Tutorial) is a great place to start.
-3.  If you want to get started making your own SVGs, you'll want some sort of visual editor.
-    1.  [This is a great little SVG editor](https://yqnn.github.io/svg-path-editor) for learning how the SVG markup works. It not only shows the XML, but also lets you see the "commands" you're using to generate the SVG. This website isn't really designed for making complex graphics, though.
-    2.  For that, an excellent option is [Inkscape](https://inkscape.org/), which is not only free but open source!
-    3.  And if you're really invested in creating your own SVGs, you might want to check out some of the powerful paid options out there, such as [Affinity Designer](https://affinity.serif.com/designer/).
-4.  If you want to use SVGs to generate images programmatically, there are at least two major, modern libraries for that: [snap.svg](http://snapsvg.io/) and [SVG.js](https://svgjs.dev/docs/3.0/).
-5.  For data visualization, [d3](https://d3js.org/) has been the standard for many, many years.
-6.  If you're interested in some of the more advanced things you can do with SVGs, check out [this video on SVG animation](https://www.youtube.com/watch?v=UTHgr6NLeEw), and this article on [SVG Filters, and Why They're Awesome](https://www.smashingmagazine.com/2015/05/why-the-svg-filter-is-awesome/)!
+This section contains helpful links to related content. It isn't required, so consider it supplemental.
+
+- There are lots of great free SVG icon libraries, such as [Material icons](https://fonts.google.com/icons) and [Feather icons](https://feathericons.com/).
+- If you want to get started making your own SVGs, you'll want some sort of visual editor.
+    1. If you wish to play around with creating or editing SVGs, check out [Yann Armelin's SVG path editor](https://yqnn.github.io/svg-path-editor). It's great for tinkering around with simpler SVGs but not designed for complex graphics.
```

Comment:
```suggestion
- If you want to get started making your own or editing SVGs, you'll want some sort of visual editor like [Yann Armelin's SVG path editor](https://yqnn.github.io/svg-path-editor). It's great for tinkering around with simpler SVGs but not designed for complex graphics.
```
In hindsight, because the AR section needed to be fixed to an unordered list instead, mind changing these two lines to be a single bullet like the above suggestion?